frame/staking-rewards/src/benchmarking.rs

```diff
@@ -98,12 +98,25 @@ benchmarks! {
 		assert_last_event::<T>(Event::StakeAmountExtended { position_id, amount}.into());
 	}
 
+	unstake {
```

Comment:
I realize that for `stake`, `unstake`, `split` operations we need to make the benchmarks parametric over the number of rewarded assets. i.e we need to measure the execution time when varying the number of rewarded assets. Would you be able to create a task for that?

Reply:
Also, It is relevant for `create_reward_pool` and `extend`.
 Fixed.